Cargo.toml

```diff
@@ -32,6 +32,7 @@ members = [
     "consensus",
     "network",
     "client",
+    "client2",
```

Comment:
When do we get rid of `client` and rename `client2` to `client`? 😆

Reply:
After `client2` supports RPC and metrics server. Feel free to implement that.